Source/WebCore/page/Quirks.cpp

```diff
@@ -1522,4 +1523,29 @@ bool Quirks::shouldExposeShowModalDialog() const
     return *m_shouldExposeShowModalDialog;
 }
 
+bool Quirks::shouldDisableLazyImageLoadingQuirk() const
+{
```

Comment:
I was planning to add a link to the current Ikea discussion.

For the quirks I think I would like a bit more information then just a link to a bugzilla. Having information on the background of why this quirk exists, and information on how to eventually remove it would be nice too. @karlcow thoughts?

Reply:
More would be better. We could come up with a kind of format where we could normalize step by step the other quirks too, but just start with this one. 
Something that can be easily parsed.